### PR TITLE
Explained how to disable the _id

### PR DESCRIPTION
**Pull request by dandv**
...+ fixed some of the English

### CODE REVIEW

**Comment by hasezoey** [comment]
your documentation is wrong, please look at https://typegoose.github.io/typegoose/docs/decorators/prop/#_id

**Comment by dandv** [comment]
That's because I tried looking at the available options for `@prop({ })` but got "No suggestions":

image

The autocomplete used to work on v6. Same IDE (WebStorm).

**Comment by hasezoey** [comment]
this probably means that webstorm uses an typescript below 3.8 (requirement for typegoose 7.0)
or its just another webstorm thing

**Comment by dandv** [comment]
It uses 3.8.3, and it used to work before. Might be a bug.

File: github-page/_guides/advanced/changeIDType.md [hunk 1]
```diff
@@ -4,20 +4,30 @@ redirect_from:
...
 class SomeChangedID {
   @prop()
-  public _id: string; // to change the _id to type of string
+  public _id: string;  // change the type of _id to string
+}
+```
+
+Note: when the type is not `ObjectID`, you need to explicitly set the `_id` before saving.
+
+To disable the `_id` field altogether (useful in arrays of subdocuments), add `@prop({ _id: false })`.
```
Comment (hasezoey) [inline github-page/_guides/advanced/changeIDType.md:18]:
```suggestion
To disable the `_id` field altogether (useful in arrays of subdocuments), add [`@prop({ _id: false })`]({{ site.baseurl }}{% link _docs/decorators/prop.md %}#_id).
```

maybe add a link to the actual option

**Comment by dandv** [comment]
I don't get useful autocomplete in VS Code either:

image

**Comment by hasezoey** [comment]
@dandv in case of vscode, check your project version of typescript, your global version (if used) and if your vscode version has integrated typescript 3.8 (1.44)
and if it still isnt working, do you have `@types/mongoose` installed?
-> if it *still* dosnt work, try reinstalling typegoose in the project, otherwise i dont know any further
(if it still dosnt work, maybe check vscode output of `Typescript`(you need to enable verbose logging))

PS: if you really use vscode in that image, and default highlighting colors, then it seems that something is wrong, because all decorators shouldnt be the same color as an value

screenshot

File: github-page/_guides/advanced/changeIDType.md [hunk 2]
```diff
@@ -4,20 +4,30 @@ redirect_from:
...
 class SomeChangedID {
   @prop()
-  public _id: string; // to change the _id to type of string
+  public _id: string;  // change the type of _id to string
+}
+```
+
+Note: when the type is not `ObjectID`, you need to explicitly set the `_id` before saving.
+
+To disable the `_id` field altogether (useful in arrays of subdocuments), add [`@prop({ _id: false })`]({{ site.baseurl }}{% link _docs/decorators/prop.md %}#_id).
```
Comment (hasezoey) [inline github-page/_guides/advanced/changeIDType.md:18]:
```suggestion
To disable the `_id` field altogether (useful in arrays of subdocuments), add [`@prop({ _id: false })`]({{ site.baseurl }}{% link _docs/decorators/prop.md %}#_id) or [`@modelOptions({ schemaOptions: { _id: false } })`](https://mongoosejs.com/docs/guide.html#_id).
```

**Comment by dandv** [comment]
VS Code also uses 3.8.3:

image

@types/mongoose is installed:

image

I've reinstalled @typegoose/typegoose, v7.0.0, same problem, no autocomplete suggestions from either VS Code or WebStorm.

**Comment by dandv** [comment]
I found the cause of the lack of autocompletion: instead of the regular `import { ... } from '@typegoose/typegoose'`, I was using this form of import, which is compatible with [Jest not fully supporting ES Modules](https://github.com/facebook/jest/issues/9430):

```
import * as typegoose from '@typegoose/typegoose';
import { Ref } from '@typegoose/typegoose';
const { index, prop, arrayProp, getModelForClass, defaultClasses } = typegoose['default'] || typegoose;

```

**Comment by dandv** [comment]
I found the cause of the lack of autocompletion: instead of the regular `import { ... } from '@typegoose/typegoose'`, I'm using this form of import, which is compatible with tsc generating ES Modules (`"module": "esnext"`) (which Typegoose doesn't support, #214):

```
import * as typegoose from '@typegoose/typegoose';
import { Ref } from '@typegoose/typegoose';
const { index, prop, arrayProp, getModelForClass, defaultClasses } = typegoose['default'] || typegoose;
```

**Pull request merged**